ZAPD/Main.cpp

```diff
@@ -35,18 +35,29 @@ void BuildAssetBlob(const fs::path& blobFilePath, const fs::path& outPath);
 #define ARRAY_COUNT(arr) (sizeof(arr) / sizeof(arr[0]))
 void ErrorHandler(int sig)
 {
-	void* array[4096];
-	const size_t nMaxFrames = sizeof(array) / sizeof(array[0]);
+	void* array[0x1000];
+	const size_t nMaxFrames = ARRAY_COUNT(array);
```

Comment:
This can just be
```suggestion
	const size_t nMaxFrames = std::size(array);
```
No need for ugly macros.